example/src/main/java/com/vimeo/android/networking/example/vimeonetworking/TestAccountStore.java

```diff
@@ -38,19 +39,18 @@ public VimeoAccount loadAccount() {
         return AccountPreferenceManager.getClientAccount();
     }
 
-    @Deprecated
     @Override
-    public void saveAccount(VimeoAccount vimeoAccount, String email, String password) {
-        saveAccount(vimeoAccount, email);
+    public void saveAccount(@NonNull VimeoAccount vimeoAccount) {
+        AccountPreferenceManager.setClientAccount(vimeoAccount);
     }
 
     @Override
-    public void saveAccount(VimeoAccount vimeoAccount, String email) {
+    public void saveAccount(@NonNull VimeoAccount vimeoAccount, @NonNull String email) {
         AccountPreferenceManager.setClientAccount(vimeoAccount);
     }
 
     @Override
-    public void deleteAccount(VimeoAccount vimeoAccount) {
+    public void deleteAccount(@NonNull VimeoAccount vimeoAccount) {
         AccountPreferenceManager.removeClientAccount();
```

Comment:
@howardrigberg Can you elaborate?

If it's a simple change (seems like it is), it's something we should probably do, no?